content/es/governance.md

```diff
@@ -0,0 +1,137 @@
+Estos individuos son identificados por el CT y su incorporación como
+Colaboradores se discute en la reunión semanal del CT.
+
+_Nota:_ Si usted realiza una contribución significativa y no se le asigna
+privilegios de commit, registre un asunto o contacte directamente a un
+miembro del CT y será tomado en cuenta en la proxima reunión del CT.
+
+Las Modificaciones de los contenidos del repositorio iojs/io.js se hacen
+sobre una base de colaboración. Cualquier persona con una cuenta de GitHub
+podrá proponer una modificación a través de solicitud de extracción y será
+considerado por los colaboradores del proyecto. Todas las solicitudes de
+extracción deben ser revisadas y aceptadas por un colaborador con la
+suficiente experiencia que sea capaz de asumir plena responsabilidad por
+el cambio. En el caso de las solicitudes de extracción propuestas por un
+colaborador existente, un colaborador adicional es requerido para
+finalizarla. Se debe buscar el consenso si participan Colaboradores
+adicionales y hay un desacuerdo en torno a una modificación particular.
+Ver _Proceso de Búsqueda de Consenso_ abajo para más detalles sobre el
+modelo de consenso utilizado para la gobernabilidad.
+
+Los colaboradores pueden optar por elevar modificaciones significativas o
+controversiales, o modificaciones que no han encontrado consenso al CT
+para su debate asignando la etiqueta ***tc-agenda*** a una solicitud
+de extracción o asunto. El CT debe servir como el árbitro final cuando se
+requiera.
+
+Para la lista actual de Colaboradores, ver el proyecto
+[README.md](./README.md#current-project-team-members).
+
+Una guía para los Colaboradores es mantenida en
+[COLLABORATOR_GUIDE.md](./COLLABORATOR_GUIDE.md).
+
+## CT Membresía
+
+Los puestos del CT no son de duración limitada. No hay un tamaño fijo del
+CT. Sin embargo, el tamaño esperado es de 6 a 12 para asegurar una
+adecuada cobertura de las áreas importantes de experiencia en
+equilibrio con la capacidad de tomar decisiones eficientemente.
+
+No hay un conjunto específico de requisitos o calificaciones para ser miembro
+del CT más allá de estas reglas.
+
+El CT puede agregar miembros adicionales al CT por una moción estándar
+del TC.
+
+Un miembro de CT puede ser removido de la CT por renuncia voluntaria, o por una
+moción estándar del CT.
+
+Cambios a la membresía del CT deberían ser publicados en la agenda y
+pueden ser sugeridos como cualquier otro punto de la agenda (ver "CT Reuniones" abajo).
+
+No más de un tercio de los miembros del CT puede estar afiliados con el
+mismo empleador. Si la remoción o renuncia de un miembro del CT o un
+cambio de empleo de un miembro del CT crea una situación en la que más de
+un tercio de las membresías del CT comparten un empleador, entonces la
+situación debe remediarse inmediatamente por la renuncia o remoción de uno
+o más miembros del CT afiliados con el empleador(es) sobre-representado.
+
+## TC Reuniones
```

Comment:
* `TC Reuniones` ---> `Reuniones del TC` . If this change is applied, a reference above need to be changed.